src/Pages/miyoungshin/Login/Login.js

```diff
@@ -1,8 +1,187 @@
+  //       email: this.state.id,
+  //       password: this.state.password,
+  //       nickname: '신코드',
+  //       mobile: '010-1174-2345',
+  //     }),
+  //   })
+  //     .then(response => response.json())
+  //     .then(result => console.log('결과: ', result));
+  // };
+
+  // handleLogin = () => {
+  //   fetch('http://10.58.1.87:8002/userapp/log', {
+  //     method: 'POST',
+  //     body: JSON.stringify({
+  //       email: this.state.id,
+  //       password: this.state.password,
+  //     }),
+  //   })
+  //     .then(response => response.json())
+  //     .then(result => console.log('결과: ', result));
+  // };
+
   render() {
-    return <div></div>;
+    const { id, password, isIdFocus, isPwFocus, isLoginActive } = this.state;
+    return (
+      <div className="loginMi">
+        <section>
+          <div className="mainImages">
+            <img
+              src="/images/miyoungshin/login/smartphone.png"
+              alt="smartphone"
+            />
+            <img
+              className="thumbnailImage"
+              src="/images/miyoungshin/login/rolling-1.jpeg"
+              alt="instagram thumbnail"
+            />
+          </div>
+        </section>
+        <section className="contents">
+          <article className="logInBox">
+            <h1>Westagram</h1>
+            <div
+              className={
+                'idInput ' + (isIdFocus ? 'inputFocusOn' : 'inputFocusOff')
+              }
+            >
+              <input
+                type="text"
+                placeholder="Phone number, username, or email"
+                name="id"
+                value={id}
+                onFocus={this.idFocusChange}
+                onBlur={this.idFocusChange}
+                onChange={this.inputChange}
+                onKeyUp={this.checkValid}
+              />
+              <div onClikck={this.inputFocus}>
+                Phone number, username, or email
+              </div>
+            </div>
+            <div
+              className={
+                'pwInput ' + (isPwFocus ? 'inputFocusOn' : 'inputFocusOff')
+              }
+            >
+              <input
+                type="password"
+                name="password"
+                placeholder="Password"
+                value={password}
+                onFocus={this.pwFocusChange}
+                onBlur={this.pwFocusChange}
+                onChange={this.inputChange}
+                onKeyUp={this.checkValid}
+              />
+              <div>Password</div>
+              <button>Show</button>
+            </div>
+            <button
+              className={
+                'logInButton ' +
+                (isLoginActive ? 'logInButtonOn' : 'logInButtonOff')
+              }
+              disabled={!isLoginActive}
+              onClick={this.handleLogin}
+            >
+              Log in
+            </button>
+            <div className="or">
+              <div className="orLine"></div>
```

Comment:
```suggestion
              <div className="orLine" />
```
self closing 태그 사용하실 수 있는 경우인것 같네요!

Reply:
수정했습니다!